### ISSUE TITLE
Backpack voided my items

### ISSUE THREAD

**Issue by CrankySupertoon01** [V0idWa1k3r/SmartBackpacks#21]
Everything that is going into my backpack is voided. I am on a server, running the latest build

**Comment by gazpachoking** [comment]
Sure it's just not invisible? Try clicking on some slots as if to pick it up again. That's what's happening on my server. #18

**Comment by mrdeadlocked** [comment]
When you view the backpack is it in your bar or on your back? I have the same issue if I put the backpack on my back and toss stuff in I can see the stuff. Take it off and open it on my bar its 'empty/invisible'. I put the backpack back on and I can view the items again.

**Comment by gazpachoking** [comment]
Hmm, haven't ever tried on my back. I've just been using it from the hotbar.

**Comment by iamsewage** [comment]
This may or may not help provide any additional helpful info, but, I've had my backpack void as well. I'm playing on a server, and it was exclusively in my inventory. I had a couple upgrades in it which remain in the backpack, however the inventory was voided (also, the inventory wouldn't show up unless I clicked on the spots where they were, at which point it would show the items, however that issue is already reported here: #18 )

Great mod <3

**Comment by V0idWa1k3r** [comment]
Yeah as far as I can see backpacks currently don't work too well on the servers. I will try to test out why that is a thing however I remember having issues with that earlier when making upgrades where the client was for some reason the desicive factor in the items inside the backpack. I might have to rework how the backpack stores items.

**Comment by TheDeviantCrafter** [comment]
I had this happen as well. I was playing on a custom modded server with version 1.0.6 of the mod.

Two days ago I discovered that I couldn't open the upgrades page of any of my modpacks. After the next server reset all of my packbacks had their items purged, and the upgrades page was fine.

Oddly enough the auto-feeder upgrade I had installed wasn't lost. The backpack was definitely empty, though. I tried clicking the slots to make sure it wasn't #18 as mentioned above.

**Comment by VPelt** [comment]
I tested this and this seems to happen if you right click to open your backpack. Then if you right click against if it's gone.

However if you press shift B and then B you get access to your items again.

Also it seems that equiped with shift-B and without it provides you with 2 separate inventory sets.
